net/tac_plus/Makefile

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (C) 2006-2019 OpenWrt.org
```

Comment:
Don't assign copyright to OpenWrt.org unless you're affiliated
https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md
"Provide an up-to-date Copyright notice or none. Copyright should not be assigned to OpenWrt unless you are explicitly requested by or working under contract to OpenWrt."